### ISSUE TITLE
Pretrained model on cityscapes

### ISSUE THREAD

**Issue by codermckee** [zhanghang1989/gluoncv-torch#12]
Do you have any pretrained model on cityscapes or mapillary?

**Comment by codermckee** [comment]
I use the models pretrained on ImageNet or voc to train mapillary dataset, but get low mIoU. so sad.

**Comment by zhanghang1989** [comment]
Will add cityscapes in GluonCV in later release.

**Comment by codermckee** [comment]
> Will add cityscapes in GluonCV in later release.

thanks for your reply. I am a beginner of semantic segmentation, and I have a problem which confused me.
I use pretrained model on ImageNet to train object detector and I can get good mAP. But it  doesn't work when it comes to semantic segmentation.
would you mind giving me some advice?  thank you very much!

**Comment by zhanghang1989** [comment]
There are many details in the implementation for reproducing SoTA result. Synchronized Batch Normalization is one of the most important key in semantic segmentation.

**Comment by zhanghang1989** [comment]
You may start with my repo https://hangzhang.org/PyTorch-Encoding/experiments/segmentation.html

**Comment by codermckee** [comment]
> You may start with my repo https://hangzhang.org/PyTorch-Encoding/experiments/segmentation.html

Thank you very much !

**Comment by XiaLiPKU** [comment]
@zhanghang1989 Can you also add the pretrained model on COCO? I see that has been published in GluonCV.
